packages/apollo-language-server/src/config.ts

```diff
@@ -93,13 +93,6 @@ export interface ServiceConfigFormat extends ConfigBase {
   localSchemaFile?: string;
 }
 
-export const DefaultServiceConfig = {
```

Comment:
Totally agree with this change, but I'm concerned about people currently depending on this behavior.

Is it possible to either:
1) deprecate this with messaging and not remove it just yet? or...
2) add some error messaging with instructions for people who currently _do_ depend on this behavior so they have an easy time resolving the issue on upgrading.

Reply:
Do you think that's necessary? The behavior isn't documented in any way currently, and IMHO shouldn't be a thing to begin with.

Reply:
That's fair, I'm ok with leaving this out 👍